### DISCUSSION TITLE
Best practice for running vhserver as a linux service

### DISCUSSION THREAD

**Discussion by Visiopaja** [GameServerManagers/LinuxGSM#3378] [Q&A] [answered]
Is there a best practice on what arguments you should use in /etc/systemd/system/vhserver.service when running vhserver as a service?

I only have some simple settings in the file:

[Unit]
Description=Starts vhserver
After=network.target

[Service]
Type=forking
User=vhserver
EnvironmentFile=/home/vhserver/service-args
ExecStart=/home/vhserver/vhserver ${ARG1}
ExecStop=/home/vhserver/vhserver ${ARG2}
Restart=on-failure

[Install]
WantedBy=multi-user.target

**Comment by romleinster** [comment, marked as answer]
The docs give a template that has worked well for me: https://docs.linuxgsm.com/configuration/running-on-boot (subbing out ts3 for vh)
